### ISSUE TITLE
Can not login with Evernote or Yinxiang Biji account

### ISSUE THREAD

**Issue by ghost** [baumgarr/Nixnote2#459]
System: Manaro x86_64
Nixnote Version: 2.0.2

It should be like this 
![haspassword](https://user-images.githubusercontent.com/12682385/45740418-ce8e9f80-bc27-11e8-8f1f-3b855dbd3c33.png)
But in my case ,it get this
![nopassword](https://user-images.githubusercontent.com/12682385/45740440-dc442500-bc27-11e8-8f7b-904c402fc6f9.png)
there is no place to type in password , press the continue button ,nothing happens ,hit cancle will pop up window says: login failes , no way to login.

I install it in repository package and the AUR ,both the save issue.
Neither the evernote accout nor the Yinxiang Biji account works !
Any ideas ?

**Comment by robert7** [comment]
TLDR: **Yinxiang BiJi should work..** (and of course Evernote server also)

As you maybe noticed, the original maintainer isn't available now.. It someway happened, that I took over.. There is updated version of the app ("[v.2.1](https://github.com/robert7/nixnote2/releases)") in [my fork](https://github.com/robert7/nixnote2). So far I did mainly stabilization/bugfixing and added few new features. 

As you may guess **I can't read single character in Chinese language** .. :)
But someway using Google Translate & a bit guess work I succeeded to register in Yinxiang BiJi - entered 3 random test notes in the web client. Then started fresh Nixnote profile - logged in .. and it worked fine. Notes synced well in both directions. Chinese characters seem to be displayed correctly.

I suggest:
1, use the latest available AppImage version [v.2.1](https://github.com/robert7/nixnote2/releases)
2, start nixnote21 (eventually delete old existing profile or start with new one - you can see where profile data are, using menu Option: Help/Data and Log Location Info) - check FAQ in my wiki
3, menu File "Add Another user" - enter your nickname and select "Yinxiang BiJi"
4, switch to new account (maybe manual restart of the app will be needed) - check if the app was minimized in systray - if yes - terminate (Quilt) & restart - verify you using new account
5, click sync
6, login, confirm authorization - see screen shoots
7, **..should work :)**

In case it doesn't, you may open issue in my fork, add log file and more detailed description. in case you post some screenshots with chinese characters please provide translation :)

I'll add screenshots in next message

**Comment by robert7** [comment]
![screen_20180920_06](https://user-images.githubusercontent.com/1135218/45840457-4687c300-bd17-11e8-87ee-b683e9589019.png)
![screen_20180920_07](https://user-images.githubusercontent.com/1135218/45840465-4982b380-bd17-11e8-8abd-e70b7aabbd46.png)
![screen_20180920_08](https://user-images.githubusercontent.com/1135218/45840471-4d163a80-bd17-11e8-8c70-77c8d011e168.png)
![screen_20180920_02](https://user-images.githubusercontent.com/1135218/45840493-56070c00-bd17-11e8-80cb-28f908f05859.png)
![screen_20180920_03](https://user-images.githubusercontent.com/1135218/45840500-599a9300-bd17-11e8-8a3b-b37874e53b21.png)
![screen_20180920_04](https://user-images.githubusercontent.com/1135218/45840506-5dc6b080-bd17-11e8-86fe-e98b2a636912.png)
![screen_20180920_05](https://user-images.githubusercontent.com/1135218/45840515-628b6480-bd17-11e8-8c37-e3fa42121f80.png)

**Comment by S0urceC0der** [comment]
There is a work around for this problem which can be found in https://blog.csdn.net/tjzhaomengyi/article/details/81015166 .

Click top left icon for yinxiangbiji, and redirect to the normal index page, then login as usual in web page. After login, a white empty page will appear and just close it. Finally click the synchronise button.
